### ISSUE TITLE
Capture consent to monthly invoicing & payment within 30 days

### ISSUE THREAD

**Issue by b-f-chan** [overture-stack/enrolment#116]
New requirement from @mbrazas in #109 (creating as separate ticket here):

* Add checkbox on the registration page that the *"PI acknowledges that their usage of the Collaboratory will be invoiced monthly, and that payment is required within 30 days."*

We want to bring payment acknowledgement upfront, since we are not implementing upfront credit card capture & validation at this time.

**Comment by b-f-chan** [comment]
Blocked by: https://github.com/overture-stack/charts/issues/2

**Comment by b-f-chan** [comment]
PR #119
